documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -673,7 +670,7 @@ For a full list of possible `<locale>` values, see the abbreviations used in the
 [[change-timezone-nonint]]
 ==== Time zone
 
-Set your local time zone, starting with the region then selecting a city, e.g. "Europe/London".
+Set your local time zone in the format `Region/City`, for example 'Europe/London'. Type a letter to jump to that letter in the list.
```

Comment:
Huh? This is the documentation for the **non-interactive** usage of `raspi-config`, so I don't think you'll be able to "Type a letter to jump to that letter in the list." :wink:

Reply:
Thanks, retconned that sentence.